EventFilter/L1TRawToDigi/src/implementations_stage2/MPUnpacker_0x1001000b.cc

```diff
@@ -0,0 +1,229 @@
+#define EDM_ML_DEBUG 1
+
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "EventFilter/L1TRawToDigi/interface/Unpacker.h"
+
+#include "L1Trigger/L1TCalorimeter/interface/CaloTools.h"
+
+#include "CaloCollections.h"
+
+namespace l1t {
+   namespace stage2 {
+      class MPUnpacker_0x1001000b : public Unpacker {
+         public:
+            virtual bool unpack(const Block& block, UnpackerCollections *coll) override;
+      };
+   }
+}
+
+// Implementation
+
+namespace l1t {
+namespace stage2 {
+   bool
+   MPUnpacker_0x1001000b::unpack(const Block& block, UnpackerCollections *coll)
+   {
+
+     LogDebug("L1T") << "Block ID  = " << block.header().getID() << " size = " << block.header().getSize() << " AMC = " << block.amc().getAMCNumber();
+
+     // check this is the correct MP
+     unsigned int amc  = block.amc().getAMCNumber();
+     unsigned int bxid = block.amc().getBX();
+     //     if( (amc-1) != (bxid-1)%9 ) return true;
+     if( (amc-1) != ((bxid-1+3)%9) ) return true;   // temporary measure!
+     LogDebug("L1T") << "Unpacking AMC " << amc << " for BX " << bxid;
+
+     auto res1_ = static_cast<CaloCollections*>(coll)->getMPJets();
```

Comment:
The packer framework makes heavy use of downcasting so that specific concrete instances can use a common packer/unpacker interface.  I'm not a big fan of downcasting but its tolerable in this case because the concrete implementations reliable know the true (derived) types they are dealing with.  We did have one problem related to bad/impossible downcasting in the packers that was fixed in a recent PR.